src/pdf/DriverOverview/DriverOverviewPdfButton.tsx

```diff
@@ -43,7 +46,32 @@ type ParcelsForDeliveryResponse =
           error: { type: ParcelsForDeliveryErrorType; logId: string };
       };
 
-type ParcelsForDeliveryErrorType = "parcelFetchFailed" | "noMatchingClient";
+type ParcelsForDeliveryErrorType = "parcelFetchFailed" | "noMatchingClient" | "eventFetchFailed";
+
+type ParcelEventResponse =
+    | {
+          data: Schema["events"][];
+          error: null;
+      }
+    | {
+          data: null;
+          error: { type: ParcelsForDeliveryErrorType; logId: string };
+      };
+
+// Gets all "shipping labels downloaded" events for the given parcels
+const getParcelEvents = async (parcelIds: string[]): Promise<ParcelEventResponse> => {
```

Comment:
Actually I think it makes sense to just put this data fetching into `getParcelsForDelivery`. We can access the events foreign table in the same call. If it's not possible to filter by the `new_parcel_status` and sort by the `timestamp` in this table in the call (it might be), we can do this in the front end.

Reply:
Yeah I couldn't figure out how to do this earlier but I think I've done it now! Lmk if it can be improved further :)